test/api/v1/manufacturers/list.test.ts

```diff
@@ -17,3 +17,13 @@ test("GET /v1/manufacturers/list (filter by integration status)", async (t: Exec
   })
   t.is(data.manufacturers.length, 0)
 })
+
+test("GET /v1/manufacturers/list (filter by liqe_query)", async (t: ExecutionContext) => {
+  const { axios } = await getTestServer(t)
+  const { data } = await axios.get("/v1/manufacturers/list", {
+    params: {
+      liqe_query: "display_name:seam",
```

Comment:
Just curious, is case insensitive the default?

Reply:
yep